CHANGELOG.md

```diff
@@ -1,6 +1,378 @@
 Changelog
 =========
 
+0.6.0
+-----
+
+**Date:** 2021-12-XX
+
+C++14, New Solvers, I/O via openPMD API, HIP Support
+
+This release switches to C++14 as minimum required version. Transition to C++17
+is planned for upcoming releases.
+
+We extended PIConGPU with a few new solvers. Binary collisions are now
+available. We added arbitrary-order FDTD Maxwell's solver. All field solvers are
+now compatible with perfectly matched layer absorber, which became default.
+There is a new way of generating incident field based on the total
```

Comment:
Maybe this should be reduced or excluded altogether? At the release, it's only for Yee (tho the laser is also only for Yee)

Reply:
Agree with excluding or explicitly say "generating incident filed based  ... for Yee field solver".

Reply:
Clarified and shortened